src/components/Footer.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import {
+  Box,
+  Footer,
+  Grid,
+  Button,
+  Image,
+  Form,
+  FormField,
+  TextInput,
+} from 'grommet';
+import { Text } from '../legos/typography/Text';
+import Logo from '../../static/assets/logo.svg';
+
+import SendButtonIcon from '../../static/assets/sendButton.svg';
+
+const linkItems = [
+  { id: '1', label: 'Work', link: '/work' },
+  { id: '2', label: 'Our team', link: '/our-team' },
+  { id: '3', label: 'Contacts', link: '/contacts' },
+];
+
+export const SiteFooter = () => (
+  <Footer background="#104065" height="350px" justify="stretch">
```

Comment:
It's better to store values that are frequently used in some place and then import them from there. For example, color values that are used often should be stored in a variable somewhere. Pass not color values but color names that are defined in `utils/theme.js`.

For example color value #104065 is hardcoded. Instead of just copy pasting it, you can just mention id as `brand` because it's defined in our theme.

Another benefit of storing these values globally, is that if we will need to change the color in the future, we won't have to rewrite it everywhere, just in that single place.